components/FinancialSummary/SponsorshipTiers.js

```diff
@@ -0,0 +1,73 @@
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-base text-[#212526]">$100/month</td>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 text-left md:text-base text-[#212526]">
+                                        Company logo in README on GitHub
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-base text-[#212526] text-left">Silver</td>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-base text-[#212526]">$500/month</td>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 text-left md:text-base text-[#212526]">
+                                        Company logo in README on GitHub and asyncapi.com
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-base text-[#212526] text-left">Gold</td>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-base text-[#212526]">$1000/month</td>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-left text-base text-[#212526]">
+                                        Company logo in README on GitHub and asyncapi.com
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-base text-[#212526]">Platinum</td>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 md:text-base text-[#212526]">$2000/month</td>
+                                    <td className="border border-gray-500 p-2 md:px-6 md:py-2 md:px-10 md:py-4 text-left md:text-base text-[#212526]">
+                                        Company logo in README on GitHub and asyncapi.com.
+                                        Up to 2 hours of support per month. Support will be
+                                        remote with the option of a shared screen or via private chat.
+                                        Support hours do not accumulate.
+                                    </td>
+                                </tr>
+                            </tbody>
+                        </table>
```

Comment:
Instead of manually hardcoding this table content, store the information in a JSON-type format and then render it using static JSON content. Kindly follow the good code-practices.